starter-code/src/AddFood.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+
+
+
+class AddFood extends Component {
+
+    state = {
+        name: '',
+        calories: '',
+        quantity: '',
+        image: ''
+    }
+
+    submitHandler = (event) => {
+        event.preventDefault()
+        this.props.onNewFood(this.state.name, this.state.calories, this.state.quantity, this.state.image)
+    }
+
+    // changeName = (event) => {
+    //     this.setState({
+    //         name: event.target.value
+    //     })
+    // }
+
+    // changePopularity = (event) => {
+    //     this.setState({
+    //         popularity: event.target.value
+    //     })
+    // }
+
+    changeHandler = (event) => {
+        const inputName = event.target.name
+        this.setState({
+            [inputName]: event.target.value
+        })
+    }
+    render() {
+
+
+
+        return (
+            <div>
+
+                <form onSubmit={this.submitHandler}>
+                    <p>Add a food:</p>
+                    <input name="name" placeholder="name"
+                        onChange={this.changeHandler}
+                        value={this.state.name}></input> <br></br>
```

Comment:
This double `<br>` is a bit weird and one would probably use css to create space between components.